Sprint-3/3-mandatory-practice/implement/get-ordinal-number.js

```diff
@@ -1,5 +1,41 @@
 function getOrdinalNumber(num) {
- return "1st";
+    const theLastDigit = num % 10;
+    const theFirstTwoDigits = num % 100;
+    
+    // We check if the number is 1, and if the number is 11, then we take the number and add the suffix "th" and return it, but if the number has any last two digits but 11, then we take the number and add the suffix "st" and return it. example: 111 => 111th, 21 => 21st
+     if (theLastDigit === 1) {
+        if(theFirstTwoDigits === 11) {
+            return num + "th";
+        }
+        else{
+            return num + "st";
+        }
+    }
+
```

Comment:
It is good to see you use nesting if check in your logic, but nesting if checks makes your code little bit complicated, try not to nest too much if else statements if it's not necessary.  You can make this code more clean if you first check numbers for 1,2,3 when you get the argument and then continue to  apply your if else logic